luminous-lightyears/src/player.py

```diff
@@ -0,0 +1,67 @@
+from typing import TYPE_CHECKING
+import time
+from attrs import define
+from interactions import Modal, ModalContext, ShortText, SlashContext
+
+if TYPE_CHECKING:
+    from src.game import Game
+
+
+@define
+class PlayerState:
+    """The current state of the player."""
+
+    nation_name: str
+
+    # Money with the government
+    money: float = 100
+
+    # How loyal people feel to the current government that you have created
+    loyalty: float = 50
+
+    # How vulnerable is the country from external threats
+    security: float = 50
+
+    # Lower means entity sabotage and vice versa (might add this as a later future)
+    world_opinion: float = 50
+
+    def apply(self, consequence: dict) -> None:
+        """Apply the consequnces to current state."""
+        for k, v in consequence.items():
+            setattr(self, k, getattr(self, k, None) + v)
+
+
+class Player:
+    def __init__(self, ctx: SlashContext, game: "Game") -> None:
+        self.ctx: SlashContext = ctx
+        self.state: PlayerState = None  # type: ignore TODO: properly type that state isn't none after register
+        self.game: Game= game
```

Comment:
This was caught by linting in the workflow https://github.com/krishnabhat3383/code-jam-24-luminous-lightyears/actions/runs/10202760910/job/28227602120#step:4:186
```suggestion
        self.game: Game = game
```